fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/ReindexWorker.java

```diff
@@ -89,6 +94,7 @@ public void run() {
                         service.cleanupSession(transactionId);
                         throw e;
                     }
+                    LOGGER.error("Reindexing of OCFL id {} failed: {}", id, e.getMessage());
```

Comment:
Can you change this to:

```java
LOGGER.error("Reindexing of OCFL id {} failed", id, e);
```

It'll make it a lot easier to figure out what went wrong